### PR TITLE
disable pagination of orders when syncing

### PR DESCRIPTION
**Pull request by josh-dadswell-hackney**

### CODE REVIEW

File: RepairsApi/V2/UseCase/ListWorkOrdersUseCase.cs [hunk 1]
```diff
@@ -33,18 +33,21 @@ public async Task<IEnumerable<WorkOrder>> Execute(WorkOrderSearchParameters sear
 
             workOrders = EnsureSorting(searchParameters, workOrders);
 
+            if (searchParameters.PageSize <= 0)
+            {
+                return workOrders;
```
Comment (Asura5) [inline RepairsApi/V2/UseCase/ListWorkOrdersUseCase.cs:38]: Are we not going to hit the lambda execution time limit with this eventually?

Comment (Asura5) [reply RepairsApi/V2/UseCase/ListWorkOrdersUseCase.cs:38]: Also, I think anything <0 should be an error state, 0 exactly being this code path

Comment (josh-dadswell-hackney) [reply RepairsApi/V2/UseCase/ListWorkOrdersUseCase.cs:38]: Hopefully not, but potentially, depends on how many work orders we end up raising in one day. How else would you recommend we handle it?

Comment (Asura5) [reply RepairsApi/V2/UseCase/ListWorkOrdersUseCase.cs:38]: Let's cross that bridge when we come to it

**Review by Asura5 — APPROVED** [review]

**Pull request merged**